plugins/inputs/couchdb/couchdb.go

```diff
@@ -128,6 +134,8 @@ func (c *CouchDB) fetchAndInsertData(accumulator telegraf.Accumulator, host stri
 	c.MapCopy(fields, c.generateFields("couchdb_request_time", stats.Couchdb.RequestTime))
 	c.MapCopy(fields, c.generateFields("couchdb_database_reads", stats.Couchdb.DatabaseReads))
 	c.MapCopy(fields, c.generateFields("couchdb_open_os_files", stats.Couchdb.OpenOsFiles))
+	c.MapCopy(fields, c.generateFields("request_time", stats.RequestTime))
+	c.MapCopy(fields, c.generateFields("open_os_files", stats.OpenOsFiles))
```

Comment:
I think we should reuse the field name from before, even though I prefer without the couchdb prefix.  It looks like the datatype is compatible.

Reply:
The only downside with using the same field name prefix is that the suffix will be different. `couchdb_request_time_value` will be the new populated field while `couchdb_request_time_current` will be populated for pre-2.0 versions.

Reply:
Yeah, your way fits the input data better.

Reply:
Then again, maybe using the same prefix is ok since for 2.0 versions, the `.._value` measurement is what matters. Which brings up another question as to what to do with the unused fields (`_mean`, `_max`, etc...) for 2.0 versions

Reply:
On second look the prefix is coming from structure of the input data, and now they have moved these fields outside of the couchdb subobject, so I think it is okay to mirror this.

On the unset fields we should make sure not to set them, looks like generateFields needs modified.